craigslist/__init__.py

```diff
@@ -65,6 +65,8 @@ class CraigslistBase(object):
         'search_titles': {'url_key': 'srchType', 'value': 'T'},
         'has_image': {'url_key': 'hasPic', 'value': 1},
         'posted_today': {'url_key': 'postedToday', 'value': 1},
+        'search_distance': {'url_key': 'searchDistance', 'value': 1},
+        'postal': {'url_key': 'postal', 'value': None},
```

Comment:
Use 'zip_code' instead of 'postal' to use the same key as the one being used in CraigslistHousing.
You should also remove search_distance + zip_code from CraigslistHousing, since this will make the filter available for all subclasses.